granary/tests/test_reddit.py

```diff
@@ -7,22 +7,25 @@
 import json
 
 class FakeRedditor():
+  # see https://praw.readthedocs.io/en/latest/code_overview/models/redditor.html
```

Comment:
very minor: docstring instead of comment pls

Reply:
ah, right, fixed.